packages/snjs/lib/services/features_service.ts

```diff
@@ -72,8 +72,9 @@ export class SNFeaturesService extends PureService<void> {
 
     this.removeExtensionRepoItemsObserver = this.itemManager.addObserver(
       ContentType.ExtensionRepo,
-      async (changed, inserted) => {
-        if (this.sessionManager.getUser()) {
+      async (changed, inserted, _discarded, _ignored, source) => {
+        const sources = [PayloadSource.Constructor, PayloadSource.LocalChanged];
```

Comment:
I think maybe Constructor and RemoteRetrieved would be what you want. Constructor would cover initial DB load event, and remote retrieved when it is retrieved from the server.

Reply:
But I’m not sure Constructor is what we want. Maybe LocalRetrieved but I have to double check what that is

Reply:
Yup LocalRetrieved is what covers initial DB load items: https://github.com/standardnotes/snjs/blob/9133cefec991f32a29511dafb2eef094dc598244/packages/snjs/lib/services/sync/sync_service.ts#L295

Reply:
Tried changing it to using those two sources, but both tests fail 😕

Reply:
Let me try running this locally..